.github/workflows/rill-cloud.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow triggers deployment with Rill Cloud helm charts
+# Each merge to main branch is build with github sha tag and published to Rill Cloud.
+name: Deploy to Rill Cloud
+on:
+  push:
+    tags:
+      - "**"
+    branches:
+      - main
+      - cd
+  workflow_dispatch:
+
+env:
+  RELEASE: ${{ contains('refs/tags', github.ref_name) }}
+
+jobs:
+  release:
+    name: Deploy to Rill Cloud
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.2
+
+      - name: Set up GCloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Authenticate GCloud
+        uses: google-github-actions/auth@v1
+        with:
+          credentials_json: "${{ secrets.RILL_BINARY_SA }}"
+
+      - name: Build & Publish Rill docker image
+        run: |-
+          go build -o rill cli/main.go
+
+          gcloud auth configure-docker
+
+          docker build -t gcr.io/rilldata/rill-headless:${GITHUB_SHA} .
```

Comment:
Noticed this uses `ubuntu:focal` – is that okay or should we do Alpine or Debian slim?

Reply:
there were some issues at the time of goreleaser.